rbi/annotations/minitest.rbi

```diff
@@ -0,0 +1,114 @@
+  def refute_empty(obj, msg = nil); end
+
+  sig { params(exp: T.anything, act: T.anything, delta: Numeric, msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
+  def refute_in_delta(exp, act, delta = T.unsafe(nil), msg = nil); end
+
+  sig { params(a: T.anything, b: T.anything, epsilon: Numeric, msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
+  def refute_in_epsilon(a, b, epsilon = T.unsafe(nil), msg = nil); end
+
+  sig { params(collection: T.anything, obj: T.anything, msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
+  def refute_includes(collection, obj, msg = nil); end
+
+  sig { params(cls: T.anything, obj: T.anything, msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
+  def refute_instance_of(cls, obj, msg = nil); end
+
+  sig { params(cls: T.anything, obj: T.anything, msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
+  def refute_kind_of(cls, obj, msg = nil); end
+
+  sig { params(exp: T.anything, act: T.anything, msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
+  def refute_equal(exp, act, msg = nil); end
+
+  sig { params(matcher: T.any(String, Regexp), obj: T.anything, msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
+  def refute_match(matcher, obj, msg = nil); end
+
+  sig { params(block: T.proc.void).returns(T::Boolean) }
+  def refute_pattern(&block); end
+
+  sig { params(o1: T.anything, op: T.any(Symbol, String), o2: T.anything, msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
+  def refute_operator(o1, op, o2 = T.unsafe(nil), msg = nil); end
+
+  sig { params(path: T.any(String, Pathname), msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
+  def refute_path_exists(path, msg = nil); end
+
+  sig { params(obj: T.anything, meth: T.any(String, Symbol), msg: T.nilable(T.any(String, Symbol, Proc)), include_all: T::Boolean).returns(T::Boolean) }
+  def refute_respond_to(obj, meth, msg = nil, include_all: false); end
+
+  sig { params(o1: T.anything, op: T.any(String, Symbol), msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
+  def refute_predicate(o1, op, msg = nil); end
+
+  sig { params(exp: T.anything, act: T.anything, msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
+  def refute_same(exp, act, msg = nil); end
+
+  sig { params(obj: T.anything, msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
+  def refute_nil(obj, msg = nil); end
+
+  sig { params(test: T.anything, msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
+  def assert(test, msg = nil); end
+
+  sig { params(obj: T.anything, msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
+  def assert_empty(obj, msg = nil); end
+
+  sig { params(exp: T.anything, act: T.anything, delta: Numeric, msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
```

Comment:
```suggestion
  sig { params(exp: T.anything, act: T.anything, delta: Float, msg: T.nilable(T.any(String, Symbol, Proc))).returns(T::Boolean) }
```
https://apidock.com/ruby/MiniTest/Assertions/assert_in_delta

Reply:
The comparisons themselves that Minitest applies are not exclusive to floats and we have quite a large number of usage of this with integers. Do you see any issues with us being less restrictive?

Reply:
I've seen it used with `BigDecimal` too, but I'm not sure if it was a correct usage (i.e. if it triggered coercion to `Float` internally, and degeated the point of `BigDecimal)